.buildkite/pipeline.yml

```diff
@@ -0,0 +1,47 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  plugins: &common_plugins
+  - &bash_cache automattic/bash-cache#v1.3.2: ~
+  # Common environment values to use with the `env` key.
+  env: &common_env
+    IMAGE_ID: xcode-12.5.1
+
+steps:
+  #################
+  # Build and Test
+  #################
+  - label: "🧪 Build and Test"
+    key: "test"
+    command: |
+      echo "--- :beer: Installing Dependencies"
+      brew bundle --file .buildkite/brewfile
+
+      echo "--- :rubygems: Setting up Gems"
+      install_gems
+
+      echo "--- :hammer: Build DrawText"
+      bundle exec rake compile
+
+      echo "--- :git: Setting up git-lfs"
+      git-lfs install
+
+      echo "--- :rspec: Run Rspec"
+      bundle exec rspec --profile 10 --format progress
+    env: *common_env
+    plugins: *common_plugins
+    agents:
+      queue: "mac"
+  #################
+  # Lint
+  #################
+  - label: "🧹 Lint"
+    command: |
+      echo "--- :rubygems: Setting up Gems"
+      install_gems
+
+      echo "--- :rubocop: Run Rubocop"
+      bundle exec rubocop
```

Comment:
This is already [handled by GitHub Actions calling Danger calling rubocop — and this doing inline comments on the PR for violations found](https://github.com/wordpress-mobile/release-toolkit/blob/develop/Dangerfile).

Personally despite inline comments being a nice feature I feel this setup has been a bit too noisy in the past (especially when we work on a new feature and add lots of code and totally forget to run rubocop before opening the PR and have lots of small, and often similar, violations).
So I'd be totally ok to have Builkite handle rubocop like you did here (with just a check failure and no inline comment on the PR) and get rid of it being run by Danger with inline comments.

Not sure if you planned to get rid of GitHub Actions and Danger in a separate PR but I'd be up for us to do it in this one?

Reply:
Yeah, my plan was to remove all of that separately, but happy to do it as part of this one 😃

Reply:
Ok 👍  whatever you prefer (separate or same PR), as long as it's already in your plans 🙂

Reply:
6a0e52b resolves this. I'm going to resolve this conversation accordingly to avoid confusion when seeing the wall of text 😄